src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -3038,7 +3038,6 @@ class DacStreamManager
 //----------------------------------------------------------------------------
 
 ClrDataAccess::ClrDataAccess(ICorDebugDataTarget * pTarget, ICLRDataTarget * pLegacyTarget/*=0*/)
-    : m_cdac{CDAC::Invalid()}
```

Comment:
If you convert this to `: m_cdac{}`, you can change the `CDAC` constructor to be `CDAC() = default;`